src/track/beatutils.cpp

```diff
@@ -35,15 +35,8 @@ const double kBpmFilterTolerance = 1.0;
 constexpr double kMaxDiffSameBpm = 0.25;
 
 bool checkIfErrorGrowing(double previous, double current) {
-
-    if (current < 0 && previous > 0) {
-        return false;
-    } else if (current > 0 && previous < 0) {
-        return true;
-    } else {
-        return fabs(current) > fabs(previous)
-                ? true : false;
-    }
+    return fabs(current) > fabs(previous)
+            ? true : false;
```

Comment:
you can omit "? true : false;"

Reply:
Right, done. No need for this function at all. Removed.